data/core/common.lua

```diff
@@ -228,8 +329,26 @@ function common.match_pattern(text, pattern, ...)
 end
 
 
+---@alias common.textalign
+---| '"left"'   # Align text to the left of the bounding box
+---| '"right"'  # Align text to the right of the bounding box
+---| '"center"' # Center text in the bounding box
+
+---Draws text onto the window.
+---The function returns the X and Y coordinates of the bottom-right
+---corner of the text.
+---@param font renderer.font
+---@param color renderer.color
+---@param text string
+---@param align common.textalign
```

Comment:
Could we integrate the type in the param field?
Like it's done in https://github.com/LuaLS/lua-language-server/wiki/Annotations#param -> Custom Type Parameter